extensions/wikia/Lightbox/templates/Lightbox_lightboxModalContent.php

```diff
@@ -123,6 +123,36 @@
 				</div>
 			</div>
 			<div class="bottom-forms">
+                <?php
+                    if(!$wg->Out->getContext()->getUser()->isAnon()) {
```

Comment:
1. There is already a property added in #16569 you could and should use.
2. Mustache templates have conditionals - use them instead of falling back to PHP (see: https://mustache.github.io/mustache.5.html)

Reply:
Different scope, but I've added something similar